src/assets/js/dynamic-addon-cards.js

```diff
@@ -139,6 +139,7 @@
 
           if (!isIncompatible && clientApp === 'android') {
             const isRecommended =
+              addon.type === 'extension' &&
```

Comment:
Nit: This isn't really part of what contributes to an add-on being Recommended. The code certainly works, but it seems slightly inaccurate. Should this test perhaps be added to the test on line 147 instead? Or perhaps change `isRecommended` to `isInstallable` or something similar?

Reply:
oh that's right.